Userland/Utilities/groups.cpp

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) 2021, the SerenityOS developers.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <grp.h>
+#include <pwd.h>
+#include <unistd.h>
+
+#include <AK/Vector.h>
+#include <LibCore/Account.h>
+#include <LibCore/ArgsParser.h>
+
+static void print_gid_list(const Vector<gid_t>& gids)
+{
+    for (auto gid : gids) {
+        auto* gr = getgrgid(gid);
+        printf("%s ", gr->gr_name);
+    }
+    printf("\n");
+}
+
+int main(int argc, char** argv)
+{
+    if (unveil("/etc/passwd", "r") < 0) {
+        perror("unveil");
+        return 1;
+    }
+
+    if (unveil("/etc/group", "r") < 0) {
+        perror("unveil");
+        return 1;
+    }
+
```

Comment:
`unveil` state should be locked once all necessary files have been unveiled.

```cpp
    unveil(nullptr, nullptr);
```

Reply:
Thanks! Change made and squashed ; should have read `man unveil` to its end :sweat_smile: